recipes/undetected-chromedriver/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "undetected-chromedriver" %}
+{% set version = "3.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/undetected-chromedriver-{{ version }}.tar.gz
+  sha256: 9f945e1435005247abe17de316bcfda85b284a4177fd5f25167c78ced33b65ec
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - selenium >=4.9.0
+    - requests
+    - websockets
+
+test:
+  imports:
+    - undetected_chromedriver
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ultrafunkamsterdam/undetected-chromedriver
+  summary: "('Selenium.webdriver.Chrome replacement with compatiblity for Brave, and other Chromium based browsers.', 'Not triggered by CloudFlare/Imperva/hCaptcha and such.', 'NOTE: results may vary due to many factors. No guarantees are given, except for ongoing efforts in understanding detection algorithms.')"
+  license: GPL-3.0-or-later
```

Comment:
Not sure if this is `GPL-3.0-or-later` or `GPL-3.0-only`.

https://github.com/ultrafunkamsterdam/undetected-chromedriver/blob/master/LICENSE

Reply:
Looks like "only"

Reply:
```suggestion
  license: GPL-3.0-only
```